### PR TITLE
versionBRS 

### PR DESCRIPTION
**Pull request by frankTheTank72**
Was in conflict with version of tx

### CODE REVIEW

**Comment by pir8radio** [comment]
LOL  damnit frank i get an email every time there is a change to this repository..   😁  😂

File: java_wallet/models.py [hunk 1]
```diff
@@ -102,11 +102,7 @@ class Asset(models.Model):
     quantity = PositiveBigIntegerField()
     decimals = models.IntegerField()
     height = models.IntegerField()
-    mintable = False
-    
-    version = os.environ.get('BRS_P2P_VERSION')
-    if version.startswith('3.3'):
```
Comment (jjos2372) [inline java_wallet/models.py:108]: v3.4 also have this, we probably need to have this differently.

File: scan/templatetags/burst_tags.py [hunk 2]
```diff
@@ -48,12 +48,8 @@ def block_fee_miner(block: Block) -> float:
 @cache_memoize(3600)
 @register.filter
 def asset_circulating(asset_id: int) -> int:
-    version = os.environ.get('BRS_P2P_VERSION')
     asset_details = BrsApi(settings.SIGNUM_NODE).get_asset(asset_id)
-
-    if version.startswith('3.3'):
```
Comment (jjos2372) [inline scan/templatetags/burst_tags.py:54]: similarly here.

**Comment by frankTheTank72** [comment]
Yes i removed the check and added     return int(asset_details["quantityCirculatingQNT"]) as result

**Pull request merged**